settings.gradle

```diff
@@ -0,0 +1,16 @@
+
+pluginManagement {
+    repositories {
+        mavenLocal()
+        maven { url "https://repo.grails.org/grails/core/" }
+        gradlePluginPortal()
+    }
+    plugins {
+        id "org.grails.grails-web" version "6.1.2"
+        id "org.grails.grails-gsp" version "6.1.2"
+//        id "com.bertramlabs.asset-pipeline" version "4.3.0"
```

Comment:
remove if not required, also check if webdriver binaries are really needed?

Reply:
Removed.